classes/patterns/class-sideimagewithtextandcta.php

```diff
@@ -31,13 +31,16 @@ public static function get_config( $params = [] ): array {
 		$media_link           = esc_url( get_template_directory_uri() ) . '/images/placeholders/placeholder-546x415.jpg';
 		$media_position_class = array_key_exists( 'media_position', $params ) && 'right' === $params['media_position'] ? 'has-media-on-the-right' : '';
 		$title_placeholder    = $params['title_placeholder'] ?? '';
+		$background_color     = $params['background_color'] ?? null;
+		$background_attribute = $background_color ? ',"backgroundColor":"' . $background_color . '"' : '';
+		$background_classes   = $background_color ? 'has-' . $background_color . '-background-color has-background' : '';
 
 		return [
 			'title'      => __( 'Side image with text and CTA', 'planet4-blocks-backend' ),
 			'categories' => [ 'planet4' ],
 			'content'    => '
-				<!-- wp:media-text {"mediaLink":"' . $media_link . '","mediaType":"image","className":"block ' . $media_position_class . '"} -->
-					<div class="wp-block-media-text alignwide is-stacked-on-mobile block ' . $media_position_class . '">
+				<!-- wp:media-text {"mediaLink":"' . $media_link . '","mediaType":"image","className":"block ' . $media_position_class . $background_classes . '"' . $background_attribute . '} -->
```

Comment:
No need to add background class in the className property it's handled by backgroundColor prop, looks like adding it would "lock" the color depending on the one you try to click on

Reply:
Very good point 👍 updated!